README.md

```diff
@@ -248,44 +162,42 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
+> Readers are encouraged to support this project by contributing.
 
-If you like this project...
+If you like this project, feel free to use it, leave a comment and
+follow me for update.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank microverse for this __wonderful__ opportunity.
+And also to Gregoire Vella on Behance who Original design inspired this idea.
```

Comment:
- [ ] While you did well adding the acknowledgments just as the first reviewer @rloterh  had requested, it would equally be great to ensure that you have included the links to the design as well. This would help people who would like to replicate the same design know where to get it when they want it.